guides/contribution.md

```diff
@@ -0,0 +1,119 @@
+# Contribution guide
+
+Welcome to the Mística Design System contribution guide for external teams.
```

Comment:
:robot: OpenAI

The introduction is clear and concise. However, it would be helpful to add a brief summary of what the contribution guide entails.
```suggestion
# Contribution Guide

This guide outlines the process for proposing, creating issues, updating or creating specifications, and moving from design to development in the Mística Design System for external teams. It also includes an index and approval criteria for proposals.
```

<!-- This is an auto-generated comment by OpenAI -->